client/src/pages/Shop.js

```diff
@@ -134,11 +30,15 @@ export default function Shop() {
     },
     isMyShop: false,
     error: null,
-    isLoading: false
+    isLoading: false,
+    shopDescriptionInput: "",
+    shopTitleInput: "",
+    editDetailsDialogIsOpen: false,
+    editCoverDialogIsOpen: false
   });
-
   // fetch shop data
   const token = localStorage.getItem("token");
+
   const USER_API = "http://localhost:3001/users/";
```

Comment:
I know this is not your code, but also here not to use http://localhost:3001

Reply:
This was my code. I have removed localhost from all API paths in shop.